tests/worksheet_test.py

```diff
@@ -1871,3 +1871,19 @@ def test_get_and_get_values_have_same_signature(self):
 
         # get_all_values should be a carbon copy of get_values
         self.assertEqual(sig_get_values, sig_get_all_values)
+
+    @pytest.mark.vcr()
+    def test_add_validation(self):
+        sheet = self.sheet
+        self.assertDictEqual(
+            sheet.add_validation(
+                "A1",
+                utils.ValidationConditionType.one_of_list,
+                "y",
+                "n",
```

Comment:
like the docs, this is also wrong now, no? it is only luck that it passes

it should be ["y", "n"]

it may also be nice to add one other validation condition to the tests, like `text_contains`, or `number_not_between`, etc, to get a bit more coverage.

(I think there is also an argument for testing each type of condition, since we *are* the ones to be providing each `enum`)... @lavigne958 ?